product_secondary_unit/models/product_second_unit.py

```diff
@@ -42,16 +42,10 @@ class ProductSecondaryUnit(models.Model):
     factor = fields.Float(string="Secondary Unit Factor", default=1.0, required=True)
     active = fields.Boolean(default=True)
 
-    def name_get(self):
-        result = []
+    @api.depends("name", "factor")  # Asegúrate de incluir todos los campos relevantes
```

Comment:
```suggestion
    @api.depends("name", "factor")
```
or write it in English, please.

Reply:
It was a reminder, removed